src/components/toolTip/ToolTip.tsx

```diff
@@ -0,0 +1,32 @@
+import clsx from "clsx";
+import _ from "lodash";
+import * as React from "react";
+import * as styles from "./ToolTip.module.css";
+
+interface ToolTipProps {
+  children: React.ReactNode;
+  tooltip: string;
+}
+
+export const ToolTip: React.FC<ToolTipProps> = ({ children, tooltip }) => {
+  const [active, setActive] = React.useState(false);
```

Comment:
Whenever having a boolean active or current state, you can best call it `isActive` instead of `active`. This way, it's easier to read.